tests/test_intertidal.py

```diff
@@ -23,11 +42,204 @@ def test_intertidal_cli():
             "2022",
             "--modelled_freq",
             "3h",
+            # "--windows_n",
+            # 50,
         ],
     )
     assert result.exit_code == 0
 
 
+@pytest.mark.dependency(depends=["test_intertidal_cli"])
+def test_dem_accuracy(

```

Comment:
Function that compares integration test results to LiDAR validation, and exports plot and readme summary files